src/ui/Folding/Folding.ts

```diff
@@ -515,6 +515,10 @@ export class Folding extends Component {
       .then((results: IQueryResults) => {
         this.handlePreprocessMoreResults(results);
         return results.results;
+      })
+      .catch(e => {
```

Comment:
It's a really small detail, but if we want to follow the Coveo TSJS guide don't we want wrap the *e* with parenthesis : `(e)` ?

Reply:
It's mostly a personal opinion, but I don't agree with those kind of "written guides somewhere in a wiki". They never work, and are never followed.

If we want to enforce styling guides like this, it needs to be enforced with tooling/linter/actual scripts/bots that check for whatever arbitrary rule we deem important.